js/json-parser.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
```

Comment:
This feels like an over-complication. Why do this:

```
_url[i] = out.response.results[i].webUrl;
_title[i] = out.response.results[i].webTitle;
_body[i] = out.response.results[i].fields.body;
_thumbnail[i] = out.response.results[i].fields.thumbnail;
```
And elsewhere this:
```
article = new Article();
article.setTitle(titles[i]);
article.setBody(bodies[i]);
article.setThumbnail(thumbnails[i]);
article.setUrl(urls[i]);
```
When you could just do this:
```
var article = new Article(results[i].webUrl, results[i].webTitle, results[i].fields.body, results[i].fields.thumbnail);
```